### PR TITLE
Refactoring neural examples

### PR DESCRIPTION
**Pull request by iraikov**
This pull request adds a couple of additional neural models, and makes all neural models part of the Sims.Examples.Neural module.

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -301,7 +301,7 @@ end
     """ ->
     function gplot(sm::SimResult)
         N = length(sm.colnames)
-        figure()
+        Gaston.figure()
```
Comment (tshort) [inline src/utils.jl]: I see you used Gaston. That must mean it still works (after your fix)! I wasn't sure.

Comment (iraikov) [inline src/utils.jl]: Yes, for some reason the call to figure needs to be fully qualified.
Actually, when I first started using Sims, Gaston was broken so I used
Winston, now it seems to be the other way around :-)

Comment (tshort) [inline src/utils.jl]: Interesting. What's broken about Winston? Also, I've been liking PyPlot lately.

**Comment by tshort** [comment]
Looks like Travis is happy, so merge away!

**Comment by tshort** [comment]
This looks like a good time to bump the version number and push that to Metadata. I probably won't get a chance to do that until next week, though. The NEWS needs to be updated, and the docs need to be reviewed and regenerated.

File: src/utils.jl (hunk 1 above)
Comment (iraikov) [inline src/utils.jl]: I don't know, just started getting some Tk errors with Winston recently.
PyPlot also has some build errors, but I have not investigated very much.

**Pull request merged**